.markdownlint.json

```diff
@@ -1,12 +0,0 @@
-{
```

Comment:
For Optimism, this file is also removed when you detach specs. (https://github.com/ethereum-optimism/optimism/pull/9121/files#diff-7acf5910ecddb0a88c58debaedb129a30ebad965e878f6ebba3dad5a4fd8758a) 
I'll keep the lint file since we have the MD file left over

Reply:
Ok, please restore it.